index.html

```diff
@@ -39,30 +62,11 @@
       </select>
     </section>
     <section id="chickens" class="tab-content">
```

Comment:
The arrangement of your filters in a separate section from the set of Chickens causes the filter drop-downs to always be displayed (incorrectly at the bottom of the About tab, and correctly at the top of the Coop tab). Take a look at the assigments we've worked on to see how to more properly arrange the filters and what the filters apply to.